### PR TITLE
Added Forest Admin integration doc

### PR DESCRIPTION
**Pull request by SeyZ**
## What kind of change does this PR introduce?
This is a documentation update for connecting Supabase to Forest Admin.

## What is the current behavior?
N/A

## What is the new behavior?
N/A

## Additional context
This is the first draft. Please feel free to add comments and we will update the doc accordingly.

### CODE REVIEW

File: apps/docs/pages/guides/integrations/forestadmin.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Step 2: Connect the database to Forest Admin
+
+First, you have to create a new project on Forest Admin:
+![forestadmin-create-project](/docs/img/guides/integrations/forestadmin/forestadmin-create-project.png)
+
+Then, you can use Forest Admin's Instant Setup for the Cloud mode to quickly get started. Alternatively, you can host the generated backend admin API on your own (Advanced setup), giving you full control of the backend code. In this guide, we will use the Cloud mode and the Instant Setup.
+![forestadmin-hosting](/docs/img/guides/integrations/forestadmin/forestadmin-hosting.png)
+
+Finally, you can enter the database credentials you obtained in Step 1 and set them in Forest Admin by using the Connection URI mode. Don't forget to replace the password in the connection string with the database password you set in Supabase. Note that if you forget it, you can always go to your Supabase Database settings and reset your database password.
+![forestadmin-db-credentials](/docs/img/guides/integrations/forestadmin/forestadmin-db-credentials)
```
Comment (gregnr) [inline apps/docs/pages/guides/integrations/forestadmin.mdx:39]:
```suggestion
![forestadmin-db-credentials](/docs/img/guides/integrations/forestadmin/forestadmin-db-credentials.png)
```

**Review by gregnr — COMMENTED** [review]
Thanks @SeyZ! Just a few suggested changes there.

Also would you mind adding this to the low-code nav menu?
https://github.com/supabase/supabase/blob/4cfb8907075d5857ad86820f62e4514d2445f040/apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts#L1113-L1121

**Comment by SeyZ** [comment]
> Thanks @SeyZ! Just a few suggested changes there.
> 
> Also would you mind adding this to the low-code nav menu?
> 
> https://github.com/supabase/supabase/blob/4cfb8907075d5857ad86820f62e4514d2445f040/apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts#L1113-L1121

Done, thanks!

**Comment by SeyZ** [comment]
Thanks for your review @gregnr! All your suggestions have been committed and I've added Forest Admin to the NavigationMenu - Low Code category.

**Review by gregnr — APPROVED** [review]

**Comment by gregnr** [comment]
@SeyZ merged! Sorry for the delay there and thanks again for the contribution!

**Pull request merged**